substrate/frame/sudo/src/extension.rs

```diff
@@ -110,6 +111,7 @@ where
 		Ok((
```

Comment:
We could also add a sufficient reference to the signer here automatically. Sounds a bit cleaner than removing `CheckNonce`.

Reply:
What you mean by this?

Reply:
We remove `CheckNonce` because `CheckNonce` requires the account to exist.

But if we put `CheckOnlySudoAccount` before `CheckNonce` and `CheckOnlySudoAccount` creates the account with `inc_sufficient`: https://github.com/paritytech/polkadot-sdk/blob/007565a7225cba59d236ffc2fdaca2438a3665c1/substrate/frame/system/src/lib.rs#L1656

Then the account exists and the nonce is checked.

Reply:
And when is `dec_sufficients` called? :D

Reply:
yes true forget it :-), he could live forever